multiqc/modules/fgbio/ErrorRateByReadPosition.py

```diff
@@ -73,33 +85,48 @@ def parse_reports(self):
         'data_labels': [
             {'name': 'Error Rate', 'ylab': 'Error Rate'},
-            {'name': 'A > C', 'ylab': 'A to C error rate'},
-            {'name': 'A > G', 'ylab': 'A to G error rate'},
-            {'name': 'A > T', 'ylab': 'A to T error rate'},
-            {'name': 'C > A', 'ylab': 'C to A error rate'},
-            {'name': 'C > G', 'ylab': 'C to G error rate'},
-            {'name': 'C > T', 'ylab': 'C to T error rate'},
-        ]
+            {'name': 'A > C', 'ylab': 'A to C error rate (and T to G when collapsed)'},
+            {'name': 'A > G', 'ylab': 'A to G error rate (and T to C when collapsed)'},
+            {'name': 'A > T', 'ylab': 'A to T error rate (and T to A when collapsed)'},
+            {'name': 'C > A', 'ylab': 'C to A error rate (and G to T when collapsed)'},
+            {'name': 'C > G', 'ylab': 'C to G error rate (and G to C when collapsed)'},
+            {'name': 'C > T', 'ylab': 'C to T error rate (and G to A when collapsed)'},
+        ],
     }
 
+    uncollapsed_labels = [
+        {'name': 'G > A', 'ylab': 'G to A error rate'},
+        {'name': 'G > C', 'ylab': 'G to C error rate'},
+        {'name': 'G > T', 'ylab': 'G to T error rate'},
+        {'name': 'T > A', 'ylab': 'T to A error rate'},
+        {'name': 'T > C', 'ylab': 'T to C error rate'},
+        {'name': 'T > G', 'ylab': 'T to G error rate'},
+    ]
+
+    if not collapse:
+        pconfig['data_labels'] += uncollapsed_labels
+
+    keys = linegraph_keys + (non_collapsed_keys if not collapse else [])
+
     # Build a list of linegraphs
-    linegraph_data = [{} for _ in linegraph_keys]
+    linegraph_data = [{} for _ in keys]
     for s_name, s_data in all_data.items():
         for read_number, read_data in s_data.items():
             s_name = "%s_R%d" % (s_name, int(read_number) + 1)
-            for lg, index in zip(linegraph_data, range(7)):
-                lg[s_name] = dict((d['position'], d[linegraph_keys[index]]) for d in read_data.values())
+            for index, lg in enumerate(linegraph_data):
+                lg[s_name] = dict((d["position"], d[keys[index]]) for d in read_data.values())
 
     # add a section for the plot
-    self.add_section (
-        name = 'Error Rate by Read Position',
-        anchor = 'fgbio-error-rate-by-read-position',
-        description = 'Error rate by read position. Plot tabs show the error rates for specific substitution types.',
-        helptext = '''
-        The error rate by read position. Substitution types are collapsed based on the
-        reference or expected base, with only six substitution types being reported:
-        `A>C`, `A>G`, `A>T`, `C>A`, `C>G` and `C>T`.
-        For example, `T>G` is grouped in with `A>C`.
+    self.add_section(
+        name='Error Rate by Read Position',
+        anchor='fgbio-error-rate-by-read-position',
+        description='Error rate by read position. Plot tabs show the error rates for specific substitution types.',
```

Comment:
Perhaps add "collapse=true/false" somwhere here instead of in the legend?  It's clear in the legend if it's collapsed or not given the # of lines.

Reply:
I added the `collapse` status to the description